llvm/include/llvm/Support/TimeProfiler.h

```diff
@@ -83,6 +83,8 @@ namespace llvm {
 
 class raw_pwrite_stream;
 
+enum class TimeTraceEventType { CompleteEvent, InstantEvent, AsyncEvent };
```

Comment:
It would be useful to add a comment explaining what different event types are about (e.g. it would mention which fields are ignored by instant events)

Reply:
Done.